docs/serving/services/http-protocol.md

```diff
@@ -50,3 +52,31 @@ You can override the default behavior for each Service or global configuration.
         network:
           http-protocol: "redirected"
     ```
+
+## HTTP/1 Full Duplex support per workload
+
+Knative services can turn on the support for [HTTP/1 full duplex](https://pkg.go.dev/net/http#ResponseController.EnableFullDuplex) end-to-end on the data path.
+This should be used in scenarios where the [related Golang issue](https://github.com/golang/go/issues/40747) is hit eg. the application server writes back to QP's reverse proxy before the latter has consumed the whole request.
+For more details on why the issue appears see [here](https://github.com/golang/go/issues/40747#issuecomment-1382404132).
+
+### Configure HTTP/1 Full Duplex support
+
+In order to enable the HTTP/1 full duplex support you can set the corresponding annotation at the service level as follows:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: default
+  annotations:
+    features.knative.dev/http-full-duplex: "Enabled"
```

Comment:
does the casing matter? eg. I assume "enabled" works?

Reply:
I haven't tested this specific case, but usually all our flags are string.EqualFold'ded.